lib/ReactViews/Custom/Chart/LineChart.tsx

```diff
@@ -0,0 +1,51 @@
+import { LinePath } from "@visx/shape";
+import { line } from "d3-shape";
+import React from "react";
+import { observer } from "mobx-react";
+import { makeObservable } from "mobx";
+
+type ItemType = any;
+
+interface LineChartProps {
+  id: string;
+  chartItem: ItemType;
+  scales: { x: (p: number) => number; y: (p: number) => number };
```

Comment:
The propTypes type was "object", is this the right TypeScript type for `scales`?